core/src/main/scala/akka/persistence/postgres/query/dao/ReadJournalQueries.scala

```diff
@@ -27,16 +25,35 @@ class ReadJournalQueries(val readJournalConfig: ReadJournalConfig) {
       persistenceId: Rep[String],
       fromSequenceNr: Rep[Long],
       toSequenceNr: Rep[Long],
-      max: ConstColumn[Long]) =
+      max: ConstColumn[Long]): Query[JournalTable, JournalRow, Seq] =
+    baseTableQuery()
+      .filter(_.persistenceId === persistenceId)
+      .filter(_.sequenceNumber >= fromSequenceNr)
+      .filter(_.sequenceNumber <= toSequenceNr)
+      .sortBy(_.sequenceNumber.asc)
+      .take(max)
+
+  private def _messagesOrderingBoundedQuery(
+      persistenceId: Rep[String],
+      fromSequenceNr: Rep[Long],
+      toSequenceNr: Rep[Long],
+      max: ConstColumn[Long],
+      minOrdering: Rep[Long],
+      maxOrdering: Rep[Long]): Query[JournalTable, JournalRow, Seq] =
     baseTableQuery()
       .filter(_.persistenceId === persistenceId)
+      .filter(_.deleted === false)
```

Comment:
No need for this clause, it's already added in `baseTableQuery`

Reply:
🎯